app/api/api.rb

```diff
@@ -0,0 +1,29 @@
+class API < Grape::API
+  format :json
+  prefix :api
+  version 'v1', :path
+
+  helpers do
+    def current_user
+      user = User.find_by(authentication_token: request.headers['Authorization'])
+      if user
+        @current_user = user
+      else
+        false
+      end
+    end
+
+    def authenticate!
+      error!('401 Unauthorized', 401) unless current_user
+    end
+
+    def host
+      Rails.env.production? ? 'morning-plains-26206.herokuapp.com' : 'http://localhost:3000'
+    end
```

Comment:
would be better if you created helpers folder and put this into host_helper.rb or something like it